portality/static/doaj/js/suggestions_and_journals.js

```diff
@@ -154,7 +154,11 @@ jQuery(document).ready(function($) {
     $("#editor_group").change(function(event) {
         event.preventDefault();
         $("#editor").html("<option val='' selected='selected'></option>")
-    })
+    });
+
+    $(".application_journal_form").bind("submit", function(event) {
```

Comment:
```.bind()``` should be ```.on()```. https://www.w3schools.com/jquery/event_bind.asp

Reply:
When I declare this event bind in the JS console on the application page it works as intended, but I see no effect when running the DOAJ. Perhaps something is superseding it in the imports?